src/Ledger/Conway/Specification/Transaction.lagda.md

```diff
@@ -31,45 +35,54 @@ open import Relation.Nullary.Decidable using (⌊_⌋)
 data Tag : Type where
   Spend Mint Cert Rewrd Vote Propose : Tag
 unquoteDecl DecEq-Tag = derive-DecEq ((quote Tag , DecEq-Tag) ∷ [])
+```
+-->
 
-\end{code}
-
-\begin{NoConway}
 Some key ingredients in the transaction body are:
-\begin{itemize}
-  \item A set \txIns{} of transaction inputs, each of which identifies an output from
-    a previous transaction.  A transaction input consists of a transaction id and an
-    index to uniquely identify the output.
-  \item An indexed collection \txOuts{} of transaction outputs.
-    The \TxOut{} type is an address paired with a coin value.
-  \item A transaction fee. This value will be added to the fee pot.
-  \item The hash \txid{} of the serialized form of the
-    transaction that was included in the block.
-\end{itemize}
-\end{NoConway}
-\begin{Conway}
-Ingredients of the transaction body introduced in the Conway era are the following:
-\begin{itemize}
-  \item \txvote{}, the list of votes for goverance actions;
-  \item \txprop{}, the list of governance proposals;
-  \item \txDonation{}, amount of \Coin{} to donate to treasury, e.g., to return money
-    to the treasury after a governance action;
-  \item \currentTreasury{}, the current value of the treasury. This field serves as a
-    precondition to executing Plutus scripts that access the value of the treasury.
-\end{itemize}
-\end{Conway}
-
-\begin{figure*}[ht]
-\begin{code}[hide]
+
++  A set `txIns`{.AgdaField} of transaction inputs, each of which
+   identifies an output from a previous transaction. A transaction input
+   consists of a transaction id and an index to uniquely identify the
+   output.
+
++  An indexed collection `txOuts`{.AgdaField} of transaction outputs. The
+   `TxOut`{.AgdaFunction} type is an address paired with a coin value.
+
++  A transaction fee, `txFee`{.AgdaField}, which will be added to the fee pot.
+
++  The hash of the serialized form of the transaction that was included
+   in the block.
+
+
+Ingredients of the transaction body introduced in the Conway era are the
+following:
+
++  `txGovVotes`{.AgdaField}, the list of votes for goverance actions;
+
++  `txGovProposals`{.AgdaField}, the list of governance proposals;
+
++  `txDonation`{.AgdaField}, the amount of `Coin`{.AgdaFunction} to donate to
+   treasury, e.g., to return funds to the treasury after a governance action;
+
++  `currentTreasury`{.AgdaField}, the current value of the treasury. This
+   field serves as a precondition to executing Plutus scripts that access
+   the value of the treasury.
+
+
+## Transaction Types
+
+```agda
 record TransactionStructure : Type₁ where
   field
-\end{code}
-\emph{Abstract types}
-\begin{code}
+```
+
+*Abstract types*
```

Comment:
Maybe make these headings into subsubsections?

Reply:
Hmmm... that might be a good idea.  I'm not sure.  We've always had these kinds of type labels, since the era of pdfs.  Of course, that doesn't mean it's the right way to present them.  But I wonder if it's best to keep record types contained in a single subsection, rather than spreading their fields across subsections. 🤔  That might be better for getting a "high level" understanding of the code. Though I suppose splitting a large record up into subsections with explanatory prose would make it easier to explain things at a "low level."

Well, I'll leave it for now and we can discuss whether improvements should be made in a future issue/PR.